### DISCUSSION TITLE
❤️ Thank you!

### DISCUSSION THREAD

**Discussion by StevenBlack** [FiltersHeroes/KADhosts#73] [General]
👋🏻 Hello again @krystian3w, @hawkeye116477, and @azet12.  (Am I forgetting someone?) 

I just want to say thank you, again, for the work you gentlemen put into maintaining this list.

I'm acutely aware that I'm downstream, and what I produce is derivative.

But downstream from me is monster. I don't fully understand why it's so monster; it is what it is 🤷🏻 

I'm thankful that you all seem very aware of this, and you're clearly working hard to keep what we've got working within my self-imposed (and vague) "works on all devices" constraint. 

Because at this rate, it may someday become "all devices" or, say, increasingly close to that.

I really appreciate everything you're doing!

**Comment by hawkeye116477** [comment]
> Am I forgetting someone?

Well, there is also our AI called @PolishRoboDog :smile:.

**Comment by StevenBlack** [comment]
@hawkeye116477 well I presumed that @PolishRoboDog is a bot, I'm sorry if that's not the case @PolishRoboDog.  Good bot!

I also presumed that you are all guys (hence "gentlemen") and that's a faux-pas to correct too, if that's not the case 😄
